### PR TITLE
Remove Ruby 1.9.3 support, add testing on 2.3/2.4

### PR DESCRIPTION
**Pull request by Evesy**
## Pull Request Checklist

RE https://github.com/sensu-plugins/sensu-plugins-feature-requests/issues/27

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose

Remove 1.9.3 as it is no longer supported.
Add testing on 2.3.0 & 2.4.1

#### Known Compatablity Issues

Breaking change for users on Ruby 1.9.3

### CODE REVIEW

File: sensu-plugins-hbase.gemspec [hunk 1]
```diff
@@ -3,11 +3,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
...
 require 'date'
 
-if RUBY_VERSION < '2.0.0'
-  require 'sensu-plugins-hbase'
-else
-  require_relative 'lib/sensu-plugins-hbase'
-end
+require_relative 'lib/sensu-plugins-hbase'
 
 # pvt_key = '~/.ssh/gem-private_key.pem'
```
Comment (eheydrick) [inline sensu-plugins-hbase.gemspec]: remove this.

File: sensu-plugins-hbase.gemspec [hunk 2]
```diff
@@ -30,7 +26,7 @@ Gem::Specification.new do |s|
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 1.9.3'
+  s.required_ruby_version  = '>= 2.0.0'
   # s.signing_key            = File.expand_path(pvt_key) if $PROGRAM_NAME =~ /gem\z/
```
Comment (eheydrick) [inline sensu-plugins-hbase.gemspec]: and this.

**Review by eheydrick — CHANGES_REQUESTED** [review]
two small cleanup items otherwise looks good.

**Review by majormoses — APPROVED** [review]
LGTM

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-hbase/versions/1.0.0

**Pull request merged**